### DISCUSSION TITLE
Add new tag?

### DISCUSSION THREAD

**Discussion by Can-Zhao** [Project-MONAI/GenerativeModels#461] [General]
Hi,

The current 0.2.2 seems outdated. When shall we add a new tag?
The important update of `use_checkpointing` in [AutoencoderKL](https://github.com/Project-MONAI/GenerativeModels/blob/bdca1bd1ea8bda4960156285f805082fff7e8686/generative/networks/nets/autoencoderkl.py#L600C7-L600C20), dramatically reduces memory usage, is really desired.

Thank you!

Best,
Can

**Comment by ericspod** [comment]
We are porting everything here into Core however we should do one final release of the project as it is now. @marksgraham we had mentioned this so when time permits we should make a release. @Warvito I don't think we had an automated release system in place and you make the wheel and pushed it to pypi yourself, do you recall what your process was? Thanks!

**Reply by Can-Zhao** [reply]
Thank you so much!

**Reply by marksgraham** [reply]
Yes i can do the release this week. I think it might be a manual release, building + pushing!

**Reply by marksgraham** [reply]
Done!
